arches/app/models/models.py

```diff
@@ -1413,6 +1413,7 @@ class ETLModule(models.Model):
     etlmoduleid = models.UUIDField(primary_key=True, default=uuid.uuid1)
     name = models.TextField()
     icon = models.TextField()
+    type = models.TextField()
```

Comment:
type is a method name in core Python. Maybe this should be changed to something like 'role'? Otherwise, I'm concerned about something like this:

```
>>> type('cat')
<class 'str'>
>>> type = 'cat'
>>> type('cat')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'str' object is not callable
```